apps-client/mobile/app/_layout.tsx

```diff
@@ -160,3 +195,81 @@ function ColorSchemeProvider({ children }: { children: React.ReactNode }) {
 
 // const useIsomorphicLayoutEffect =
 //   Platform.OS === 'web' && typeof window === 'undefined' ? React.useEffect : React.useLayoutEffect
+
+function NotificationTokenConsentPopup() {
+  const registerNotificationTokenMutation = reactQueryClient.useMutation(
+    'post',
+    '/notifications/register',
+    {}
+  )
+  const user = useAuthMe()
```

Comment:
if you want to check for user's session
please use `const session = useSession` instead